package.json

```diff
@@ -34,7 +34,8 @@
     "format": "sewing-kit format",
     "type-check": "sewing-kit type-check",
     "test": "sewing-kit test",
-    "test:coverage": "yarn test --coverage",
+    "test:coverage": "CI=true yarn test --coverage",
```

Comment:
I can see value in also allowing this command to test specific files since it takes a few minutes to generate the report for all of them. I believe you can do this now with `yarn test Modal --coverage`.

Reply:
Actually, scratch that! This command works fine if you include the file name at the end like `CI=true yarn test --coverage Card`.

Reply:
Yes, args get passed through so still possible to do `yarn test:coverage card` or `dev test:coverage card`